README.md

```diff
@@ -10,6 +10,13 @@ TBD
 
 ## Development
 
+### Environment Setup
+
+#### Fedora
+```bash
+sudo dnf install clang clang-* systemd-devel
```

Comment:
Aren't following packages enough?

```
dnf install clang-tools-extra gcc make systemd-devel
```

Reply:
Exactly what I wanted to achieve with this PR :)